src/main/java/com/cleanroommc/groovyscript/compat/mods/tinkersconstruct/Casting.java

```diff
@@ -110,35 +116,44 @@ public boolean removeByCast(IIngredient cast) {
             return false;
         }
 
+        @MethodDescription(description = "groovyscript.wiki.removeAll")
         public void removeAll() {
             TinkerRegistryAccessor.getTableCastRegistry().forEach(this::addBackup);
             TinkerRegistryAccessor.getTableCastRegistry().forEach(TinkerRegistryAccessor.getTableCastRegistry()::remove);
         }
 
+        @MethodDescription(description = "groovyscript.wiki.streamRecipes")
         public SimpleObjectStream<ICastingRecipe> streamRecipes() {
             return new SimpleObjectStream<>(TinkerRegistryAccessor.getTableCastRegistry()).setRemover(this::remove);
         }
 
         public class RecipeBuilder extends AbstractRecipeBuilder<ICastingRecipe> {
 
+            @Property
             private IIngredient cast;
+            @Property(defaultValue = "200", valid = @Comp(value = "1", type = Comp.Type.GTE))
             private int time = 200;
+            @Property(defaultValue = "false")
```

Comment:
default value of a boolean is `false`, so assigning it a default value is unneeded